client/components/Modal/PageCreateModal.tsx

```diff
@@ -88,6 +88,18 @@ const PageCreateModal: FC<Props> = ({ crowi, fade = false, toggle, ...modalProps
   const [portalName, setPotalName] = useState<string>(t('Memo'))
   const [pageName, setPageName] = useState('')
   const [underTreePath, setUnderTreePath] = useState(decodeURI(parentPath(currentPath)))
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  useEffect(() => {
+    if (!modalProps.isOpen) return
+
+    // Input is not mounted immediately after isOpen changed true
```

Comment:
Why don't you try `onOpen` callback?
https://reactstrap.github.io/components/modals/

```
// called when done transitioning in
 onOpened: PropTypes.func,
```

Reply:
Fixed!